testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -126,3 +132,24 @@ def test_cosine_content():
     dot = pca_random.transform(rand.atoms)
     content = cosine_content(dot, 0)
     assert_almost_equal(content, .99, 1)
+
+def test_mean_shape(pca_aligned, u):
+    atoms = u.select_atoms(SELECTION)
+    assert_equal(pca_aligned.mean.shape[0], atoms.n_atoms * 3)
+
+def test_mean(pca_aligned, u):
+    v = mda.Universe(PSF, DCD, in_memory=True)
+    a = align.AlignTraj(v, v, select=SELECTION).run()
+    coords = v.trajectory.timeseries(v.select_atoms(SELECTION), order='fac')
+    assert_almost_equal(pca_aligned.mean, coords.mean(axis=0).ravel(), decimal=5)
+
+def test_alignment(pca_aligned, u):
+    v = mda.Universe(PSF, DCD, in_memory=True)
+    a = align.AlignTraj(v, v, select=SELECTION).run()
+    pca_pre_align = PCA(v, select=SELECTION, align=False).run()
+    assert_almost_equal(pca_aligned.mean, pca_pre_align.mean)
+    assert_almost_equal(pca_aligned.cov, pca_pre_align.cov)
```

Comment:
Thanks for the feedback @orbeckst, much appreciated.
I think this is the kind of test @lilyminium requested, ie
- comparison of covariance matrix of align=True PCA to align=False PCA on prealigned trajectory.
However I may have misinterpreted, please let me know if I have.

Reply:
Your interpretation is right, thanks @hmacdope! But I think it’s a stronger test if you don’t transfer `u` into memory before running PCA with `align=True`.